R/await.R

```diff
@@ -112,7 +112,7 @@ await <- function(f, ...,
 #' @param .y a vector of values to be passed to \code{f} (default \code{NULL})
 #' @export
 #' @describeIn await Call a function repeatedly for all values of a vector until all have reached a completed status
-await_all <- function(f, .x, .y = NULL, ...,
+await_all <- function(f, .x, .y = NULL,
```

Comment:
`.x` and `.y` should be the first and second arguments to `f`, that is, we pass the arguments by position (see also purrr::map2). The names have to be generic because the first two arguments to a poller are not always `id` `run_id`. See `workflows_get_executions` for an example. This is why `await` uses `...` to begin with.